tests/e2e/testfiles/values.yaml

```diff
@@ -1,4 +1,19 @@
 # List of E2E Tests Scenarios
+- scenario: "Install Authorization Proxy Server"
+  paths:
+    - "testfiles/authorization-templates/csm_authorization_proxy_server.yaml"
+  modules:
+    - "authorizationproxyserver"
+    - "authorization"
+  steps:
+    - "Given an environment with k8s or openshift, and CSM operator installed"
+    - "Create [authorization-proxy-server] prerequisites from CR [1]"
+    - "Apply custom resource [1]"
+    - "Validate [authorization-proxy-server] module from CR [1] is installed"
+    - "Configure authorization-proxy-server for [powerscale]"
+    - "Configure authorization-proxy-server for [powerflex]"
+    - "Delete custom resource [1]"
+
```

Comment:
This test was initially at the bottom. Felt it was important enough to get it run before the rest of the scenarios which depend on it.